lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -2062,13 +2062,10 @@ definitions:
       enable_header_enrichment:
         description: True if header enrichment feature should be disabled for gateway
         type: boolean
-        example: false
-        x-nullable: false
-        default: true
+        example: true
       enable_encryption:
         description: True if headers should be encrypted
         type: boolean
-        x-nullable: false
         example: true
       encryption_key:
         description: Key to be used in header encryption
```

Comment:
Can we all HE parameters  nullable?

Reply:
I recall not nullable makes config building simpler

Reply:
Lets improve API in 1.5